packages/components/src/focal-point-picker/styles/focal-point-picker-style.ts

```diff
@@ -21,16 +21,28 @@ export const MediaWrapper = styled.div`
 
 export const MediaContainer = styled.div`
 	align-items: center;
-	box-shadow: 0 0 0 1px rgba( 0, 0, 0, 0.2 );
+	border-radius: ${ CONFIG.radiusBlockUi };
 	cursor: pointer;
-	display: inline-flex;
+	display: flex;
 	justify-content: center;
 	margin: auto;
 	position: relative;
 	height: 100%;
 
+	&:after {
```

Comment:
Probably better to add `pointer-events: none` to this `after` pseudoelement.

Reply:
Added https://github.com/WordPress/gutenberg/pull/58459/commits/1b477c6e0f4e4ca114a06b3b4868f4899b6a2612